examples/01_plotting/plot_3d_map_to_surface_projection.py

```diff
@@ -64,11 +64,32 @@
 # computation time, but finer visualizations.
 
 big_fsaverage = datasets.fetch_surf_fsaverage('fsaverage')
-texture = surface.vol_to_surf(localizer_tmap, big_fsaverage.pial_right)
+big_texture = surface.vol_to_surf(localizer_tmap, big_fsaverage.pial_right)
 
-plotting.plot_surf_stat_map(big_fsaverage.infl_right, texture, hemi='right',
+plotting.plot_surf_stat_map(big_fsaverage.infl_right,
+                            big_texture, hemi='right',
                             title='Surface right hemisphere: fine mesh',
                             threshold=1., bg_map=big_fsaverage.sulc_right)
 
 
 plotting.show()
+
+
+##############################################################################
+# 3D visualization in a web browser
+# ---------------------------------
+# An alternative to :func:`nilearn.plotting.plot_surf_stat_map` is to use
+# :func:`nilearn.plotting.view_surf` or
+# :func:`nilearn.plotting.view_img_on_surf` that give more interactive
+# visualizations in a web browser. See :ref:`interactive-surface-plotting` for
+# more details.
+
+view = plotting.view_surf(fsaverage.infl_right, texture, threshold='90%',
+                          bg_map=fsaverage.sulc_right)
+# uncomment this to open the plot in a web browser:
+# view.open_in_browser()
```

Comment:
We should add a cell here "view", and end the cell, so that by default something running this in a notebook gets the view.

This makes me realize that maybe we should work on a good "__repr__" for the view object that says that in a notebook it appears as an interactive viewer. Conveying this notion of "__repr__" and how to trigger it is important for users.

Reply:
> This makes me realize that maybe we should work on a good "repr" for the view object that says that in a notebook it appears as an interactive viewer. Conveying this notion of "repr" and how to trigger it is important for users.

I'm not sure this is really the role of __repr__, which is more for logging and debugging. I improved the docstring so people will get these informations with view? in ipython, or by reading the docstring

Reply:
OK. But still, the example ran by default in the notebook should trigger the view. It is important for discoverability. We should not expect users to read docstrings (human nature!)

Reply:
I agree, I have added cells to display the view in both examples